zerver/tests/test_auth_backends.py

```diff
@@ -3883,13 +3883,6 @@ def test_with_redirect_to_param_set_as_next(next: str='') -> HttpResponse:
         res = test_with_redirect_to_param_set_as_next('https://rogue.zulip-like.server/login')
         self.assertEqual('http://zulip.testserver', res.url)
 
-        # In SSO based auth we never make browser send the hash to the backend.
-        # Rather we depend upon the browser's behaviour of persisting hash anchors
-        # in between redirect requests. See below stackoverflow conversation
-        # https://stackoverflow.com/questions/5283395/url-hash-is-persisting-between-redirects
-        res = test_with_redirect_to_param_set_as_next('#narrow/stream/7-test-here')
-        self.assertEqual('http://zulip.testserver', res.url)
-
```

Comment:
(This test never made sense. Since `test_with_redirect_to_param_set_as_next` had incorrectly failed to encode `#`, the `#narrow/stream/7-test-here` was parsed not as the `next` parameter but as a fragment on the outer URL, and was therefore ignored on the *client* side.)